File: app/src/main/java/com/arny/pik/models/Planing.java
package com.arny.pik.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Planing {

    @SerializedName("srcLayout")
    @Expose
    private String srcLayout;

    public String getSrcLayout() {
        return srcLayout;
    }

    public void setSrcLayout(String srcLayout) {
        this.srcLayout = srcLayout;
    }

}

File: app/src/main/java/com/arny/pik/models/Planing.kt
package com.arny.pik.models

import com.arny.arnylib.utils.Utility
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Planing {
    @SerializedName("srcLayout")
    @Expose
    var srcLayout: String? = null

    override fun toString(): String {
        return Utility.getFields(this)
    }
}